pkg/kubelet/cpumanager/policy_static_test.go

```diff
@@ -67,20 +68,34 @@ func TestStaticPolicyStart(t *testing.T) {
 func TestStaticPolicyRegister(t *testing.T) {
 	testCases := []staticPolicyTest{
 		{
+			description:     "GuPodSingleCore, SingleSocketHT, ExpectError",
+			topo:            topoSingleSocketHT,
+			numReservedCPUs: 1,
+			containerID:     "fakeID2",
+			stAssignments:   map[string]cpuset.CPUSet{},
+			stDefaultCPUSet: cpuset.NewCPUSet(0, 1, 2, 3, 4, 5, 6, 7),
+			pod:             makePod("8000m", "8000m"),
+			expErr:          fmt.Errorf("not enough cpus available to satisfy request"),
+			expCPUAlloc:     false,
+			expCSet:         cpuset.NewCPUSet(),
+		},
+		{
 			description:     "GuPodSingleCore, SingleSocketHT, ExpectAllocOneCPU",
 			topo:            topoSingleSocketHT,
+			numReservedCPUs: 1,
 			containerID:     "fakeID2",
 			stAssignments:   map[string]cpuset.CPUSet{},
-			stDefaultCPUSet: cpuset.NewCPUSet(1, 2, 3, 4, 5, 6, 7),
+			stDefaultCPUSet: cpuset.NewCPUSet(0, 1, 2, 3, 4, 5, 6, 7),
```

Comment:
Might want to add CPU 0 to the default cpuset in all test cases since we expect all CPUs in the default cpuset. This was left over from the earlier implementation of CPU manager, where we just reserved CPU 0 without looking a reserved CPU. (Note: I made this change as a part of https://github.com/ConnorDoyle/kubernetes/pull/20.)

Reply:
yep

Reply:
in some cases we should still test the allocation with some CPUs taken out of the shared set -- makes sense though that we should expect `0` to be in the shared pool always now.

Reply:
updated